cmd/runtimetest/main.go

```diff
@@ -370,6 +371,62 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
+func getActualCgroupPath() (string, error) {
+	f, err := os.Open("/proc/self/cgroup")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+
+	cgroupPath := ""
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+		if err := s.Err(); err != nil {
+			return "", err
+		}
+
+		text := s.Text()
+		parts := strings.Split(text, ":")
+		if cgroupPath == parts[2] && cgroupPath != "" {
+			continue
+		} else if cgroupPath == "" && parts[2] != "" {
```

Comment:
I think `parts[2] == ""` is not possible.  If controller was not mounted, there would not be item in /proc/self/cgroup